src/vsphere_cpi/Gemfile.lock

```diff
@@ -110,4 +109,4 @@ DEPENDENCIES
   vsphere-automation-vcenter (~> 0.1.0)
 
 BUNDLED WITH
-   2.1.4
+   1.17.2
```

Comment:
Use chruby to use ruby 2.6.5 and latest bundler. This change should not be there